ProjectFinderApi/V1/Gateways/UsersGateway.cs

```diff
@@ -35,5 +36,10 @@ public IEnumerable<User> GetUsers()
         {
             return _databaseContext.Users;
         }
+
+        public User GetUserByEmail(string email)
+        {
+            return _databaseContext.Users.Where(user => user.Email.ToLower() == email.ToLower()).FirstOrDefault();
```

Comment:
Can user.Email ever be null here? If not, then this is fine

Reply:
`user.Email` should never be null. All users must have a email to successfully insert into the database